docs/src/introduction.md

```diff
@@ -4,9 +4,48 @@ Welcome to the Polka Storage project book. This document is a work in progress a
 
 This project aims to build a native storage network for Polkadot.
 
-We've now completed Phase 2 and have started work on Phase 3.
+We've now completed Phase 3 and have started work on Phase 4.
 
-**During [**Phase 2**](https://polkadot.polkassembly.io/referenda/1150), we have implemented:**
+## **During [**Phase 3**](https://polkadot.polkassembly.io/referenda/1541), we have implemented:**
+
+### **Polka-Storage**
+
+- **Proof Systems and Performance**
+  - Enabled Groth16 batch proof verification using on-chain randomness.
+  - Productionized 1 GiB sector sizes for PoRep and PoSt.
+  - Adjusted Windowed PoSt to be partition-aware and compatible with block production time.
+  - Fine-tuned runtime constants for realistic mainnet timings in sealing and proof submission.
+
+- **Deal Automation**
+  - Introduced on-chain deal parameters published by storage providers via the market pallet.
+  - Implemented server-side deal parameter checks for automated deal validation.
+
+- **P2P Networking**
+  - Added Peer ID to Multiaddr resolution using Kademlia for peer discovery, with Multiaddr info served via libp2p request/response.
```

Comment:
Sorry, your correction made me think of this. It's borderline too technical, but technical people would find it weird that we're serving the Kad info over RR
```suggestion
  - Added Peer ID to Multiaddr resolution using Kademlia for peer discovery, with Multiaddr info served to browser clients via libp2p request/response.
```